R/time_event.R

```diff
@@ -38,6 +42,7 @@ step_time_event <-
            trained = FALSE,
            rules = list(),
            columns = NULL,
+           keep_original_cols = TRUE,
```

Comment:
good job aligning with `step_holiday()`. I like you to set the default to `FALSE`. I don't like the default that `step_holiday()` has and would like to change it long term 😄

Reply:
Will do! Funnily enough the default of `FALSE` in `step_holiday` &`step_date` always sort of made sense to me.